presto-main/src/test/java/com/facebook/presto/cost/TestScalarStatsCalculator.java

```diff
@@ -92,6 +104,41 @@ public void testLiteral()
                 .nullsFraction(1.0);
     }
 
+    @Test
+    public void testFunctionCall()
+    {
+        Slice testString = Slices.utf8Slice("ala ma kota");
+        FunctionCall fromBase64 = new FunctionCall(QualifiedName.of("from_base64"), ImmutableList.of(new StringLiteral(VarbinaryFunctions.toBase64(testString).toStringUtf8())));
+        Signature signature = FunctionRegistry.getMagicLiteralFunctionSignature(VARBINARY);
```

Comment:
`testFunctionCall` should use a simpler function call, like `concat`  or something

you should add another method like `testVarbinaryConstant` which would construct the expression using LiteralEncoder. Because -- the point is _not_ to support magic literals. It is to support whatever LiteralEncoder produces.